tests/features/custom_page/custom_page.view.feature

```diff
@@ -21,8 +21,9 @@ Feature:
     Then I should see the "About places" tile
     But I should not see the "About means" tile
 
-    # Collection facilitators can see the content created inside their collections.
+    # Collection facilitators can see the content created inside their collections
+    # but not in the search because the content are not indexed.
     When I am logged in as a facilitator of the "Drafty things" collection
     And I enter "About" in the header search bar and hit enter
     Then I should see the "About places" tile
-    And I should see the "About means" tile
+    And I should not see the "About means" tile
```

Comment:
This is because the test was looking for the item in the search page. After this ticket, the search page should only show items indexed in the published index.
I will extend the test so that the user checks as well the collection overview page and finds it.